packages/explorer/src/components/Home.js

```diff
@@ -1,11 +1,128 @@
 import React, { Component } from 'react';
+import Paper from '@material-ui/core/Paper';
+import Link from '@material-ui/core/Link';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import styles from '../css/home.module.css';
+
+import { withStyles } from '@material-ui/core/styles';
+
+const HeadingTypography = withStyles({
+  root: {
+    padding: '1rem',
+  },
+})(Typography);
+
+const SubTypography = withStyles({
+  root: {
+    paddingLeft: '1rem',
+    paddinRight: '1rem',
+    paddingBottom: '1rem',
```

Comment:
I believe presentation and layout concerns should be separated in CSS.

It seems like the entire content area could do with a wrapping element which applies the lateral gutters and then to apply a `margin` below each each text. I'm not sure if there are other MUI layout components or more global configurations we can set to define these rules?

I also see that, in addition to the spacing at the bottom of each `Typography` that we're also applying a `br` for additional spacing. We should avoid the use of `br` here.